templates/kubernetes/_get_ebook.html

```diff
@@ -5,7 +5,7 @@ <h2>A CTO’s guide to Kubernetes</h2>
 
   <div class="row">
     <div class="col-6">
-      <p class="u-hide--small"><img class="p-image--shadowed" src="https://assets.ubuntu.com/v1/daf4f278-k8s-ebook-cover-crop.jpg?w=502" width="502" alt="Cover of whitepaper"></p>
+      <p class="u-hide--small"><img class="p-image--shadowed" src="https://assets.ubuntu.com/v1/daf4f278-k8s-ebook-cover-crop.jpg?w=502" width="400" alt="Cover of whitepaper"></p>
```

Comment:
Might as well set `?w=400` in the src if we only want the image to be 400w

Reply:
Thanks. Fixed.